<launch>
	<arg name="rviz" default="true" />
	<node pkg="generalp" type="findInterestSure" name="findInterestSure" output="screen">
        <param name="featureSize" value="1.00705" />
        <param name="featureSamplingRate" value="0.891352" />
        <param name="normalsScale" value="0.793091" />
        <param name="normalSamplingRate" value="1.07546" />
        <param name="minimumCornerness" value="0.10869" />
        <param name="inputCloud" value="/velodyne_cloud_2" />
        <param name="outputCloud" value="/velodyne_cloud_3" />
	</node>
	<node pkg="velodyne_pointcloud" type="cloud_node" name="cloud_node">
		<param name="calibration" value="$(find anestis_loam_mod)/config/vlp16.yaml"/>
		<param name="min_range" value="0.4"/>
		<param name="max_range" value="135.0"/>
	</node>


  <node pkg="anestis_loam_mod" type="An_laserOdometry" name="laserOdometry" output="screen" respawn="true"/>


  <node pkg="anestis_loam_mod" type="An_laserMapping" name="laserMapping" output="screen" respawn="true"/>
  <node pkg="anestis_loam_mod" type="An_transformMaintenance" name="transformMaintenance" output="screen" respawn="true"/>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find anestis_loam_mod)/rviz_cfg/loam_velodyne.rviz" />
  </group>
  <node pkg="tf" type="static_transform_publisher" name="static_transform_publisher3" args="0 0 0 0 0 1.5  world camera_init 50"/>
<param name="use_sim_time" value="true" />

</launch>
